contracts/0.4.24/StETH.sol

```diff
@@ -81,6 +81,34 @@ contract StETH is IERC20, Pausable {
      */
     bytes32 internal constant TOTAL_SHARES_POSITION = keccak256("lido.StETH.totalShares");
 
+    /**
+      * @notice An executed shares transfer from `sender` to `recipient`.
+      *
+      * @dev emitted in pair with an ERC20-defined `Transfer` event.
+      */
+    event TransferShares(
+        address indexed from,
+        address indexed to,
+        uint256 sharesValue
+    );
+
+    /**
+     * @notice An executed `burnShares` request
+     *
+     * @dev Reports simultaneously burnt shares amount
+     * and corresponding stETH amount.
+     * The stETH amount is calculated just before the burning incurred rebase.
+     *
+     * @param account holder of the burnt shares
+     * @param amount amount of burnt shares (expressed in stETH just before burning invocation)
```

Comment:
```
@param amount amount of stETH the burnt shares corresponded to before the burn
```

Reply:
@Psirex [suggested](https://github.com/lidofinance/lido-dao/pull/401#discussion_r874516829) to use here the actual balance difference instead, wdyt?


cc: @ujenjt